runtime/oti/j9nonbuilder.h

```diff
@@ -4183,6 +4183,7 @@ typedef struct J9MemoryManagerFunctions {
 #if defined(J9VM_GC_FINALIZATION)
 	int  ( *j9gc_finalizer_startup)(struct J9JavaVM * vm) ;
 	void  ( *j9gc_finalizer_shutdown)(struct J9JavaVM * vm) ;
+	UDATA ( *j9gc_wait_for_reference_processing)(struct J9JavaVM * vm) ;
```

Comment:
(struct J9JavaVM *vm) - I see that you followed existing pattern. However I would change it any way